src/open_inwoner/utils/context_processors.py

```diff
@@ -55,7 +55,7 @@ def settings(request):
         },
         "footer": {
             "logo": config.footer_logo,
-            "logo_alt": config.name,
+            "logo_alt": config.logo.default_alt_text,
```

Comment:
@pi-sigma Not sure I am using the correct approach here... Not quite sure why this value is set differently for the footer anyway 
(would this just be for emailtemplates?)